trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/search-input-validator.test.ts

```diff
@@ -0,0 +1,126 @@
+import { add } from 'date-fns';
+import { validateSearchInput } from './search-input-validator';
+
+describe('controllers/utilisation-reports/find-reports-by-year/search-input-validator', () => {
+  const BANK_ID_ONE = '1';
+  const BANK_ID_TWO = '2';
+  const BANK_ID_THREE = '3';
+  const BANK_IDS: string[] = [BANK_ID_ONE, BANK_ID_TWO, BANK_ID_THREE];
+
+  describe('search-input-validator', () => {
+    it('returns empty errors if no bank and year', () => {
+      // Arrange
+
```

Comment:
I'm guessing empty `// Arrange` is here to make it clear that you are not including a bank or year?